google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/sql_objects/assembled_sql_objects_entry_factory.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.datacatalog_connectors.commons import prepare
+
+from google.datacatalog_connectors.rdbms.scrape import constants
+
+from google.datacatalog_connectors.rdbms.prepare.sql_objects\
+    import datacatalog_sql_objects_entry_factory,\
+    datacatalog_sql_objects_tag_factory
+
+
+class AssembledSQLObjectsEntryFactory:
```

Comment:
I wonder whether it makes sense to suppress `SQLObjects` from the class name to keep it simple since it is part of the package name. If it's not possible, I believe it's better to use `SQLObjectsAssembledEntryFactory` to keep it consistent with the names adopted in the `scrape` package. This suggestion applies not only to this but also to the other classes belonging to this package. WDYT?